pages/search/index.js

```
@@ -100,6 +100,31 @@ export async function getServerSideProps(context) {
+    for (const [key, val] of Object.entries(query)) {
+      if (key.startsWith("amp;")) {
+        const cleanKey = key.replace(/^(amp;)+/, "");
+        if (!(cleanKey in cleaned)) cleaned[cleanKey] = val;
+      } else {
+        cleaned[key] = val; // non-amp keys always win over amp-prefixed equivalents
+      }
+    }
+    const params = new URLSearchParams();
+    for (const [key, val] of Object.entries(cleaned)) {
+      if (Array.isArray(val)) {
+        val.forEach((v) => params.append(key, v));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)